source/src/behaviour/cs_BehaviourStore.cpp

```diff
@@ -16,8 +16,33 @@
 
 #include <algorithm>
 
-#define LOGBehaviourStoreInfo LOGi
-#define LOGBehaviourStoreDebug LOGd
+#define LOGBehaviourStoreInfo LOGvv
```

Comment:
I still don't like those verbosity levels in the code. Is this the level it should be? Is this temporary? From what I understand it's not seen as an issue https://github.com/crownstone/bluenet/issues/124 but I think it's a bit sloppy to commit those developer settings back and forth into the code.

Reply:
Yeah, might be nice if this would be factored into machine-local configs rather than versioned code. Put the issue into pivotal and it will happen according to prio. Maybe good to wait until cmake files have settled down a little :-)

Reply:
Wrote a small logging feature which uses VERBOSITY to determine when to output.